astropy_helpers/tests/test_setup_helpers.py

```diff
@@ -309,8 +309,6 @@ class B(A):
             run_setup('setup.py', ['build_docs', '-l'])
         elif mode == 'deprecated':
```

Comment:
Nitpick: Do we want to rename this mode to something not "deprecated" since it is no longer deprecated?

Reply:
good point, I haven't noticed it.